packages/merkle-tree/README.md

```diff
@@ -0,0 +1,48 @@
+# @ergoplatform/merkle-tree [![npm](https://badgen.net/npm/v/@ergoplatform/merkle-tree)](https://www.npmjs.com/package/@ergoplatform/merkle-tree) 
+
+This package provides functionality for working with merkle trees.
+
+Merkle trees are commonly used in distributed systems and blockchain applications to verify the integrity of data that is stored or transmitted across multiple nodes.
+
+## Installation
+
+For nodejs and bundler based frontend projects this package can be installed via NPM:
+
+```sh
+npm install @ergoplatform/merkle-tree
+```
+
+It can also be used in pure browser based code like so:
+
+```html
+<body>
+    <script type="module" src="https://cdn.jsdelivr.net/npm/@ergoplatform/merkle-tree@latest/dist/browser/ergo_merkle_tree_wasm.min.js"></script>
+
+    <script type="module">
+        import init, {LevelNode, MerkleProof} from 'https://cdn.jsdelivr.net/npm/@ergoplatform/merkle-tree@latest/dist/browser/ergo_merkle_tree_wasm.min.js';
```

Comment:
`jsdelivr` is a free CDN service for open source JS projects, basically zero config + minified browser JS for us, super useful